spring-boot-project/spring-boot/src/main/java/org/springframework/boot/logging/logback/SpringPropertyAction.java

```diff
@@ -46,6 +47,10 @@ protected Model buildCurrentModel(SaxEventInterpretationContext interpretationCo
 		model.setSource(attributes.getValue(SOURCE_ATTRIBUTE));
 		model.setScope(attributes.getValue(SCOPE_ATTRIBUTE));
 		model.setDefaultValue(attributes.getValue(DEFAULT_VALUE_ATTRIBUTE));
+
+		if (ActionUtil.stringToScope(model.getScope()) == ActionUtil.Scope.CONTEXT) {
+			interpretationContext.getContext().putProperty(model.getName(), model.getSourceOrDefaultValue());
```

Comment:
I wanted to use an appropriate Util Class in `logback`, but I couldn't find it.
`ActionUtil, ModelUtil` were not applicable in this situation.